owncloud/core
# occ command line tool to encrypt all files

occ command line tool to encrypt all files, just call without any parameters

```
./occ encryption:encrypt_all
```
- [x] Need to add unit tests. But I already tested it to a large extend manually. So feel free to have a look, test and start reviewing it while I will write the remaining unit tests

Steps to test:
- setup ownCloud
- create a nice setup (multiple users, shared files/folders, external storages,... make it as complex as you wish)
- enable default encryption module and server side encryption in the admin settings
- if you have external storages now you can decide in the settings which one should be encrypted and which not
- logout
- run the occ command
